control-plane/helper/controller/controller.go

```diff
@@ -62,7 +62,7 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 	// Add an event handler when data is received from the informer. The
 	// event handlers here will block the informer so we just offload them
 	// immediately into a workqueue.
-	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+	_, err := informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
This now returns an error and linting demands that we check it. I didn't want to change behavior so I just have it log the error.